### PR TITLE
Update profile page

### LINKED ISSUE #10: [2pts]  Profile Settings page
- Create User setting routes
- user can update hes email & password

### PR DESCRIPTION
**Pull request by aimalamiri**
## Add profile update form on the profile page

closes #10

### CODE REVIEW

**Review by d2ndjim — APPROVED** [review]
@aimalamiri, Good Job 👍 
### APPROVED ✔️

**Review by ichala — APPROVED** [review]
✔️ Approved , good job

File: src/Pages/Users/Profile.jsx [hunk 1]
```diff
@@ -1,9 +1,148 @@
...
+
+  const handleChange = (e) => {
+    setForm({ ...form, [e.target.name]: e.target.value });
+    setError(null);
+  };
+
+  const submit = (e) => {
+    e.preventDefault();
+    console.log(form.password);
+    console.log(form);
```
Comment (EidHachem) [inline src/Pages/Users/Profile.jsx]: Kindly remove console logs from the submit function

File: src/Pages/Users/Profile.jsx [hunk 2]
```diff
@@ -1,9 +1,148 @@
...
+    if (form.password !== form.confirm_password) {
+      setError('Make sure the password and password confirmation fields are the same.');
+    } else {
```
Comment (EidHachem) [inline src/Pages/Users/Profile.jsx:43]: [Optional] After the user submits the form, it is better to clear the password fields, and maybe redirect him to the home page

**Review by EidHachem — CHANGES_REQUESTED** [review]
Hi  @aimalamiri 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
169786037-9f62f867-cec1-49f6-bd13-810daaadc22d



## Highlights
- Good commit ✅ 
- All linters passed ✅ 


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the this thread if something is not 100% clear.

File: src/Pages/Users/Profile.jsx (hunk 1 above)
Comment (aimalamiri) [reply src/Pages/Users/Profile.jsx]: Nice catch! It's removed now.

File: src/Pages/Users/Profile.jsx (hunk 2 above)
Comment (aimalamiri) [reply src/Pages/Users/Profile.jsx:43]: It was a nice idea, I solved the issue.

**Review by EidHachem — APPROVED** [review]

**Review by ichala — APPROVED** [review]
✔️ Approved

**Pull request merged**